src/qmapshack/mouse/line/IScrOptEditLine.ui

```diff
@@ -304,6 +304,16 @@
        </layout>
       </widget>
      </item>
+     <item>
+      <widget class="QPushButton" name="pushOptimize">
+       <property name="toolTip">
+        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Optimizes the route by reordering the points using the router specified in the &amp;quot;Routing&amp;quot; tab. &lt;/p&gt;&lt;p&gt;This is known as solving the Traveling Salesman Problem, however start and end are kept fixed here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
You took the wrong editor. There is one for html and one for plain text. Please use the one for plain text.

If you want extended formatting create a string in the cpp file and add it as tooltip.